lib/matplotlib/backends/backend_svg.py

```diff
@@ -317,7 +317,7 @@ def _check_is_iterable_of_str(infos, key):
 
 class RendererSVG(RendererBase):
     def __init__(self, width, height, svgwriter, basename=None, image_dpi=72,
-                 *, metadata=None):
+                 *, metadata=None, aria=None):
```

Comment:
I think this is a safe extension of the API as we only use this additional optional key in this file where we know which class we are going to be calling.   I do not know if other file formats also support aria roles (as they are more of an html thing and tend to live on the DOM outside of what ever we generate).  If it is more general, we can push to other backends and up to base, otherwise I think we should keep this slight extension local to the svg backend.

Reply:
We should consider ripping this out.